Pear/Controller/TabPageViewController.swift

```diff
@@ -15,7 +15,7 @@ protocol TabDelegate: AnyObject {
 class TabPageViewController: UIPageViewController {
 
     // MARK: - Private View Vars
-    private let communityViewController = CommunityViewController()
+    private let communityViewController : UIViewController
```

Comment:
Delete space before `:`, i.e.

```suggestion
    private let communityViewController: UIViewController
```